assets/js/base/context/cart-checkout/shipping/index.js

```diff
@@ -61,33 +57,11 @@ export const useShippingDataContext = () => {
 };
 
 /**
- * Calculating component for shipping rates.
- *
- * //@todo this is currently invalid because it's based on an older api.
- * // this will need to be updated once shipping rate work is complete.
+ * The shipping data provider exposes the interface for shipping in the
+ * checkout/cart.
  *
- * @param {Object} props Incoming props for the component
+ * @param {Object} props Incoming props for provider
  */
-const ShippingRateCalculation = ( { onChange } ) => {
-	// @todo, it'd be handy if we could pass through callbacks that are fired on
-	// successful rate retrieval vs callbacks fired on unsuccessful rates
-	// retrieval. That way emitters could just be fed into the hook directly.
-	const { shippingRates, shippingRatesLoading } = useShippingRates( [
-		'country',
-		'state',
-		'city',
-		'postcode',
-	] );
-	useEffect( () => {
-		onChange( shippingRates, shippingRatesLoading );
-	}, [ shippingRates, shippingRatesLoading ] );
-	return null;
-};
```

Comment:
This component was initially implemented before the shipping address refactor and now it's obsolete so I've removed it and moved `useShippingRates` into the context provider.